src/tonstakers.ts

```diff
@@ -414,6 +422,50 @@ class Tonstakers extends EventTarget {
     }
   }
 
+  async activeWithdrawalNFTs(): Promise<NftItemWithEstimates[]> {
```

Comment:
this method should have "get" in the name

Reply:
done